types/jsdom/index.d.ts

```diff
@@ -37,20 +37,23 @@ declare module 'jsdom' {
 		serialize(): string;
 
 		/**
-		 * The nodeLocation() method will find where a DOM node is within the source document, returning the parse5 location info for the node.
+		 * The nodeLocation() method will find where a DOM node is within the source document,
+		 * returning the parse5 location info for the node.
+		 *
+		 * @throws {Error} If the JSDOM was not created with `includeNodeLocations`
 		 */
 		nodeLocation(node: Node): ElementLocation | null;
 
 		/**
-		 * The built-in `vm` module of Node.js is what underpins jsdom's script-running magic.
+		 * The built-in `vm` module of Node.js is what underpins JSDOM's script-running magic.
 		 * Some advanced use cases, like pre-compiling a script and then running it multiple
 		 * times, benefit from using the `vm` module directly with a jsdom-created `Window`.
 		 *
-		 * @throws
+		 * @throws {TypeError}
 		 * Note that this method will throw an exception if the `JSDOM` instance was created
 		 * without `runScripts` set, or if you are using JSDOM in a web browser.
 		 */
-		getInternalVMContext(): DOMWindow;
+		getInternalVMContext(): Context;
```

Comment:
The internal VM context may eventually stop being a `Window`: <https://github.com/nodejs/node/pull/30709>.